src/module-1/weirdString/index.js

```diff
@@ -1 +1,7 @@
-export const weirdString = (str = "") => {};
+export const weirdString = (str = "") => {
+
+  return str.toUpperCase().replace(/[A-Z]\b/g, function (value) {
```

Comment:
```suggestion
  return str.toUpperCase().replace(/[A-Z]\b/g, (value) => (...));
```  
consider to use arrow func here for consistency